doc/classes/AnimationNodeStateMachineTransition.xml

```diff
@@ -12,16 +12,22 @@
 	</methods>
 	<members>
 		<member name="advance_condition" type="String" setter="set_advance_condition" getter="get_advance_condition">
+			Turn on auto advance when this condition is set. This is a custom text field that can be filled with a variable name. The variable can be modified from code.
 		</member>
 		<member name="auto_advance" type="bool" setter="set_auto_advance" getter="has_auto_advance">
+			Turn on the transition automatically when this state is reached. This works best with [code]SWITCH_MODE_AT_END[/code].
 		</member>
 		<member name="disabled" type="bool" setter="set_disabled" getter="is_disabled">
+			Don't use this transition during [method AnimationNodeStateMachinePlayback.travel] or [member auto_advance].
 		</member>
 		<member name="priority" type="int" setter="set_priority" getter="get_priority">
+			When travelling from a state to another, give more priority to this node. Used together with the [method AnimationNodeStateMachinePlayback.travel] from code.
```

Comment:
`Used together with the [method AnimationNodeStateMachinePlayback.travel] from code.`
auto advance is also affected by priority. maybe it's a good idea to mention that.

Reply:
Thanks @guilhermefelipecgs, could you explain how? I couldn't find that in any of the docs. I'm guessing maybe you can actually have multiple `auto_advance` transitions and control which one is used by adjusting `priority`?

Reply:
> I'm guessing maybe you can actually have multiple `auto_advance` transitions and control which one is used by adjusting `priority`?

This is right.

See this example:
![Captura de tela de 2019-04-08 12-43-48](https://user-images.githubusercontent.com/1387165/55737693-10343b80-59fc-11e9-81e8-e5af7586bf6f.png)
![Captura de tela de 2019-04-08 12-44-05](https://user-images.githubusercontent.com/1387165/55737695-10ccd200-59fc-11e9-982c-6b7556c78c06.png)
![Captura de tela de 2019-04-08 12-43-58](https://user-images.githubusercontent.com/1387165/55737694-10343b80-59fc-11e9-9389-0774d49ed18a.png)
I need to change the priority of `Idle_2_Run -> Run` to `2` otherwise the `Die` will never play.
![Peek 08-04-2019 12-55](https://user-images.githubusercontent.com/1387165/55738545-b3398500-59fd-11e9-8af7-8009538aaae5.gif)
I've a lot of this situation in my game.

Reply:
Thanks! Do travel/auto_advance prefer a low or high priority? From your screencap it seems like lower is preferred, but it is hard to tell.

Reply:
If you have a transition with priority `1` and a second transition with priority `2`, the first transition with priority `1` will be used.